api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -66,14 +66,10 @@ message TlsParameters {
   //   [ECDHE-RSA-AES128-GCM-SHA256|ECDHE-RSA-CHACHA20-POLY1305]
   //   ECDHE-ECDSA-AES128-SHA
   //   ECDHE-RSA-AES128-SHA
-  //   AES128-GCM-SHA256
-  //   AES128-SHA
```

Comment:
@envoyproxy/api-shepherds although this change "breaks" the API, I'm not sure if there's any way around changing the default supported servers in the code otherwise.
Do you consider this to be a breaking change?

Reply:
cc @ggreenway for thoughts on this. I think we need to have the ability to change defaults like this for security reasons so this is an acceptable case. I would probably runtime guard this change though.

Reply:
We've gone through several other changes to these defaults without runtime guards.

I think in general for this, the people that have specific needs about which ciphers are supported don't use the defaults, and the new proposed defaults work for almost all use cases (the exception being very old clients).

I think if anything, we should just add a note to the docs saying "these will change over time in response to security considerations; if you care, configure it instead of using the default"

Reply:
Sure that sounds fine to me.

Reply:
This is not an API breaking change FWIW - we allow clients to freely change the defaults that they accept.

Reply:
I was under the impression that changes to default values for wrapped types are allowed, after reading the following:

> Note that changes to default values for wrapped types, e.g. google.protobuf.UInt32Value are not governed by the 
> above policy. Any management server requiring stability across Envoy API or implementations within a major version > should set explicit values for these fields.

The current field, however, is of type repeated string.

I think that adding the note @ggreenway suggested in [this comment](https://github.com/envoyproxy/envoy/pull/20862#discussion_r857818478) is the right way to go.

Reply:
I have added the note @ggreenway suggested to the docs.